solutions/instances/README.md

```diff
@@ -9,7 +9,8 @@ This architecture creates the observability instances on IBM Cloud and supports
 - Supports creating KMS root keys in an existing KMS instance or using existing keys if creating new buckets.
 - Creates a KMS encrypted COS bucket that is required to store archived logs or using an existing bucket.
 - Creates a KMS encrypted COS bucket for setting up Activity Tracker event routing or using an existing bucket.
-- Creates the Activity Tracker event route to a COS target.
+- Configures an Activity Tracker event routing to a COS bucket.
+- Optionally configures an Activity Tracker event routing to a Log Analysis instance.
```

Comment:
`Configures an Activity Tracker event routing to a COS bucket.` -> Configures the Activity Tracker event routing to a COS bucket.
`Optionally configures....` -> Supports configuring the Activity Tracker event routing to the Log Analysis instance.

Reply:
resolved